### PR TITLE
=doc Consolidate client host level docs #1290

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #1290
Update Java signature to `Pair`

### CODE REVIEW

File: docs/src/main/paradox/scala/http/client-side/host-level.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Host-Level Client-Side API
 
-As opposed to the @ref[Connection-Level Client-Side API](connection-level.md#connection-level-api) the host-level API relieves you from manually managing individual HTTP
+As opposed to the @ref[Connection-Level Client-Side API](connection-level.md) the host-level API relieves you from manually managing individual HTTP
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/host-level.md]: 👍, 5bcbd7552128881d4e665f045eea2987bf11c28d didn't remove all use of the `connection-level-api{,-java}` anchor.

File: docs/src/main/paradox/scala/http/client-side/host-level.md [hunk 2]
```diff
@@ -56,13 +57,20 @@ the same pooled connections.
...
 ```
+@@@
+@@@ div { .group-java }
+```java
+Flow<Pair<HttpRequest, T>, Pair<Try<HttpResponse>, T>, HostConnectionPool>
+```
+@@@
 
-This means it consumes tuples of type `(HttpRequest, T)` and produces tuples of type `(Try[HttpResponse], T)`
+This means it consumes tuples of type @scala[`(HttpRequest, T)`]@java[`Pair<HttpRequest, T>`] and produces tuples of type @scala[`(Try[HttpResponse], T)`]@java[`Pair<Try<HttpResponse>, T>`]
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/host-level.md]: IMO, "pairs" sounds better here than "tuples", but might be general enough.

Comment (johanandren) [inline docs/src/main/paradox/scala/http/client-side/host-level.md]: I agree, pairs is better, but I think tuples is good enough as well, so do as you please.

Comment (jlprat) [inline docs/src/main/paradox/scala/http/client-side/host-level.md]: I'll replace with pairs, as 2 people think the same way, and the same thought crossed my mind

**Pull request merged**